Tools/PIDML/pidOnnxModel.h

```diff
@@ -203,60 +193,72 @@ struct PidONNXModel {
     }
   }
 
+  static constexpr float kTOFMissingSignal = -999.0f;
+  static constexpr float kTOFPCut = 0.5f;
```

Comment:
I would prefer user-defined pT cuts, a configurable array. mPTLimits transferred here from the PidOnnxInterface.